src/System.Net.Requests/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap-Windows_NT;
```

Comment:
It will definitely use the right binaries at runtime, and while those dependencies mean that we do want to have a special configuration for System.Net.Requests implementation for uap, the tests should remain as netstandard as @danmosemsft mentions. The only reason why we would want to add a new configuration for tests, is if we want to actually produce a different test assembly (removing some tests by not compiling them, testing API only available in uap, etc.)